### PR TITLE
Update SODA documentation for Madagascar release

### PR DESCRIPTION
**Pull request by joseph-v**
**What type of PR is this?**

Update SODA documentation for Madagascar release

> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

**Review by skdwriting — APPROVED** [review]
LGTM

**Review by skdwriting — APPROVED** [review]
LGTM

**Review by anvithks — CHANGES_REQUESTED** [review]
There is already a PR (#187) to add the Lamu API spec.
Please check.

File: content/api-specs/madagascar/_index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Madagascar Release API Specification
+menuTitle: Madagascar (v1.7.0)
+description: "API Specification for the Madagascar release (v1.7.0) of various SODA Projects"
+weight: 5
```
Comment (anvithks) [inline content/api-specs/madagascar/_index.md:5]: @joseph-v This should be 3. the lighter weight bubbles up in the menu. We put the latest release at the top. The Lamu release API spec has weight 4.

**Review by anvithks — CHANGES_REQUESTED** [review]
Please check the weight of the menu. other than that LGTM

File: content/api-specs/madagascar/_index.md (hunk 1 above)
Comment (joseph-v) [reply content/api-specs/madagascar/_index.md:5]: Done

**Review by anvithks — APPROVED** [review]
LGTM

**Review by skdwriting — APPROVED** [review]
LGTM

**Pull request merged**